portal/patient_self_checkin/mySQL_Connector.php

```diff
@@ -0,0 +1,13 @@
+<?php
+$ignoreAuth = true;
+require_once "../../interface/globals.php";
+require_once $GLOBALS['webserver_root'] . '/sites/default/sqlconf.php';
```

Comment:
What is the OpenEMR approved way for connecting to the database? I searched high and low for documentation...